lib/apis/v2/felixconfig.go

```diff
@@ -187,8 +187,8 @@ type FelixConfigurationSpec struct {
 }
 
 type ProtoPort struct {
-	Protocol string
-	Port     uint16
+	Protocol string `json:"protocol"`
+	Port     uint16 `json:"port"`
```

Comment:
How does this work if people don't want to specify the port, just the protocol?

Reply:
I think we want `omitempty`

Reply:
oh ok - didn't realize that was a thing.  okey dokey - i'll update.

Reply:
i assume that's just for ports.

do we need to be able to specify ranges of ports or multiple ports in the same thing or just put this back as a string that we'll have to validate or???

Reply:
@matthewdupre @caseydavenport ^^^